test/helpers/clinics_helper_test.rb

```diff
@@ -0,0 +1,5 @@
+require 'test_helper'
+
+class ClinicsHelperTest < ActionView::TestCase
```

Comment:
If it's helpful for purposes of testing, you could rewire the clinics helper to be more 'functional' -- e.g. take a patient object as an argument and spit out the attributes as a result. But if it makes more sense to roll as is, also cool!